docs/az/docs/features.md

```diff
@@ -0,0 +1,203 @@
+# Xüsusiyyətlər
+
+## FastAPI xüsusiyyətləri
+
+**FastAPI** sizə aşağıdakıları təmin edir:
+
+### Açıq standartlara əsaslanır
+
+* API yaratmaq işlərində <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI </strong></a> buna <abbr title="also known as: endpoints, routes">path </abbr> <abbr title=" HTTP metodları olaraq bilinən, POST, GET, PUT, DELETE">əməliyyatları, </abbr>parametrləri, body tələbi, təhlükəsizlik kim şeylər daxildir.
+* Avtomatik olaraq data modelinin <a href="http://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema </strong></a> ile birlikdə sənədləşməsi(OpenAPI özü onsuz JSON Schema əsasındadır).
+* Diqqətli iş nəticəsində yuxarıda göstərilən standartlara uyğun bir framework yaratdıq. Standartları tortun üzərinə sonradan əlavə edilən çiyələk kimi görmədik.
+* Bu həmçinin bir çox dildə istifadə oluna biləcək **client code generator** istifadəsinə də icazə verir.
+
+### Avtomatik sənədləşmə
+
+OpenAPI standartlarına əsaslanan bir framework olaraq, developerlərin birdən çox seçimi var, defolt olaraq gələn 2 fərqli interaktiv API sənədləşməsi və veb istifadəçi interfeysi var.
+
+* <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank"><strong>Swagger UI </strong></a> sizə interaktiv olaraq brauzerdən API-ya çağırış etməyə və sınaqdan keçirməyə imkan verir.
```

Comment:
"çağırış etməyə" -> "sorğu göndərməyə"

Reply:
[Swagger UI] , with interactive exploration, call and test your API directly from the browser. Burda call,  çağırış kimi də istifadə etmək olar.